shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/Linker.kt

```diff
@@ -387,20 +422,23 @@ class MingwLinker(targetProperties: MingwConfigurables)
 
     override fun filterStaticLibraries(binaries: List<String>) = binaries.filter { it.isWindowsStaticLib || it.isUnixStaticLib }
 
-    override fun provideCompilerRtLibrary(libraryName: String): String? {
+    override fun provideCompilerRtLibrary(libraryName: String, isDynamic: Boolean): String? {
         val targetSuffix = when (target) {
             KonanTarget.MINGW_X64 -> "x86_64"
             else -> error("$target is not supported.")
         }
         val dir = File("$absoluteLlvmHome/lib/clang/").listFiles.firstOrNull()?.absolutePath
-        return if (dir != null) "$dir/lib/windows/libclang_rt.$libraryName-$targetSuffix.a" else null
+        val extension = if (isDynamic) "so" else "a"
```

Comment:
Should it really be `so`, not `dll`?

Reply:
I just removed support for `isDynamic` for both mingw and linux, since it's unused (and thus untested).